README.md

```diff
@@ -9,6 +9,14 @@ The Node.js project uses an [open governance model](./GOVERNANCE.md). The
 
 **This project has a [Code of Conduct][].**
 
+In addition to the code of conduct requirements, contributors to the
+Node.js project are expected to act in a collaborative manner to move
```

Comment:
```suggestion
Contributors are expected to act in a collaborative manner to move
```

Reply:
I understand the desire for brevity, but one way of reading your suggested alternative is that the code of conduct is for non-contributors and this paragraph describes the expectations of contributors. The “In addition to the code of conduct” part makes it clear that we expect contributors to abide by both.

Reply:
Other suggestion: put this paragraph _before_ the link to the CoC, so we get rid of the ambiguity and can use the shorter version.

Reply:
Feel free to iterate on my suggestion or reject it. It should not hold up landing, in my opinion.

Reply:
> Other suggestion: put this paragraph _before_ the link to the CoC, so we get rid of the ambiguity and can use the shorter version.

The Code of Conduct applies to everyone interacting with our project or repo in any way, whereas this paragraph applies only to contributors, so the CoC should probably be mentioned first.

I’m not sure how I would iterate on this to improve it; I assume the goal with this suggestion was just to shorten the text? I don’t know what I could take out without making things unclear or ambiguous.